OptionSelector/OptionsTableViewController.m

```diff
@@ -0,0 +1,126 @@
+//
+//  OptionsTableViewController.m
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OptionsTableViewController.h"
+#import "CategoryModel.h"
+#import "CQCategory.h"
+
+@interface OptionsTableViewController ()
+
+@end
+
+@implementation OptionsTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+}
+
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+
+    // Return the number of sections.
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    // Return the number of rows in the section.
+    return [self.category.options count];
+}
+
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"OptionsCellIdentifier" forIndexPath:indexPath];
+
+    cell.textLabel.text = [self.category.options objectAtIndex:indexPath.row];
+    return cell;
+}
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath   *)indexPath
+{
+    [tableView cellForRowAtIndexPath:indexPath].accessoryType = UITableViewCellAccessoryCheckmark;
+    CategoryModel *shared = [CategoryModel sharedInstance];
+    NSString *picked = [self.category.options objectAtIndex: indexPath.row];
+    
+    self.category.selection = picked;
+    
+    [shared.allCategories replaceObjectAtIndex:self.category.index withObject:self.category];
```

Comment:
I'm pretty sure you can remove this line and the `index` property from your `CQCategory` class.